emulators/qemu/Portfile

```diff
@@ -169,7 +181,7 @@ pre-configure {
 # disable silent rules
 build.args-append       V=1
 
-default_variants        +usb
+default_variants        +ssh +usb
```

Comment:
Why? I do not think the average user will ever need this...

Another reason libssh was in a variant was the OpenSSL dependency that prevented binary archives. But I guess with openssl3 this shouldn't be an issue any longer.

Reply:
I guess you're right: reverted.